force_sensor_node:
  ros__parameters:
    verbose: false
    serial_port: "/dev/ttyUSB0"
    baudrate: 57600
